test/ooni/templates.cpp

```diff
@@ -97,10 +97,10 @@ TEST_CASE("dns query template works as expected with system engine") {
                     REQUIRE(answers.is_array());
                     REQUIRE(answers.size() == 2);
                     REQUIRE((answers[0]["ttl"].is_null()));
-                    REQUIRE((answers[0]["hostname"] == "server-nexa.polito.it"));
+                    REQUIRE((answers[0]["hostname"] == "vm-nexa.polito.it"));
```

Comment:
Do we maybe want to use some other host that we control for these tests?

Reply:
Yes, using Nexa is becoming unreliable.

Reply:
Created issue at https://github.com/measurement-kit/measurement-kit/issues/1885